package.json

```diff
@@ -76,9 +76,9 @@
     "pivottable": "^2.15.0",
     "plotly.js": "1.41.3",
     "prop-types": "^15.6.1",
-    "react": "^16.3.2",
+    "react": "^16.8.3",
     "react-ace": "^6.1.0",
-    "react-dom": "^16.3.2",
+    "react-dom": "^16.8.3",
```

Comment:
Worth extracting the package changes to a separate PR, no?

Reply:
Sure. Must decide if we actually wanna tho.

Reply:
Is there a reason not to upgrade regardless of hooks?

Reply:
No